### DISCUSSION TITLE
Automod now shows messages containing blocked words

### DISCUSSION THREAD

**Discussion by CyberDruga** [Chatterino/chatterino2#5692] [Ideas] [answered]
Recently Twitch's Automod now displays when it catches messages containing blocked words, as shown in this print 
![image](https://github.com/user-attachments/assets/9f578fe2-1232-4021-9ce9-999247e3ebcd)

I didn't know if this was worthy of creating a full blown issue, but I still wanted to let you guys know :+1:

**Comment by iProdigy** [comment, marked as answer]
See https://github.com/Chatterino/chatterino2/pull/5690

**Reply by CyberDruga** [reply]
oh, I didn't search for pull requests, only issues. My bad.

**Reply by Felanbird** [reply]
It was reported via discord so we didn't bother opening an issue, just adding the opt-out they requested
